apps.json

```diff
@@ -110,6 +110,12 @@
         "state": "working",
         "url": "https://github.com/YunoHost-Apps/archivist_ynh"
     },
+    "armadietto": {
+        "branch": "master",
+        "revision": "HEAD",
```

Comment:
Please consider though that using HEAD implies you're working on a testing branch, and release only stable work on master.

Reply:
Testing branch created!
Branche testing créée !

Reply:
Thanks